app/views/errorMessage.scala.html

```diff
@@ -1,7 +1,9 @@
 @import views.html.helper._
 @(elements: FieldElements)(implicit messages: Messages)
+
 @if(elements.hasErrors) {
-    <span id="error" class="govuk-error-message">
+
+    <span id="@elements.id-error" class="govuk-error-message">
```

Comment:
Where does `@elements.id-error` come from? I don't see it in the [Play docs](https://www.playframework.com/documentation/2.8.0/api/scala/views/html/helper/FieldElements.html).

Reply:
Ah, this is just string interpolation; @element.id + the string "-error" so that the id will end up as "publicRecord-error" for example. Was not sure whether it should be "public-record-error" (like on the GDS) , publicRecordError or publicRecord-error (like what was already there) though.